assets/batch_score/components/driver/src/batch_score_oss/aoai/scoring/aoai_response_handler.py

```diff
@@ -27,11 +28,11 @@
 class AoaiHttpResponseHandler(HttpResponseHandler):
     """Defines the AOAI HTTP response handler."""
 
-    DEFAULT_MAX_RETRIES = 3
     RETRIABLE_STATUS_CODES = [408, 429]
 
-    def __init__(self, tally_handler: TallyFailedRequestHandler):
+    def __init__(self, tally_handler: TallyFailedRequestHandler, configuration: Configuration):
         """Initialize AoaiHttpResponseHandler."""
```

Comment:
Please consider adding doc strings

Reply:
added for bunch of files. param info is missing for quite a lot. Will it be okay to add in next set of PRs?